main.py

```diff
@@ -97,6 +102,15 @@ def update_db(
     status_code,
     status_api_endpoint,
 ):
+
+    total_embedding_cost = 0.0
+    total_embedding_tokens = 0
+    total_llm_cost = 0.0
+    total_llm_tokens = 0
```

Comment:
Should we consider adding these values as None when not available? Same question applies for the return from calculate_cost_and_tokens also

Reply:
Yes, I think that make sense when the value are not available. Instead of displaying 0, None will be a better option to display. I'll make the changes right away.